packages/create-instance/create-scoped-slots.js

```diff
@@ -36,25 +36,12 @@ function getVueTemplateCompilerHelpers (): { [name: string]: Function } {
 }
 
 function validateEnvironment (): void {
-  if (window.navigator.userAgent.match(/PhantomJS/i)) {
-    throwError(
-      `the scopedSlots option does not support PhantomJS. ` +
-        `Please use Puppeteer, or pass a component.`
-    )
-  }
   if (vueVersion < 2.5) {
```

Comment:
If template tag in scopedSlots mounting options works normally,
I think this limitation is not necessary.
I added this limitation since template tag in scopedSlots mounting options did not work.

https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots

> In 2.5.0+, slot-scope is no longer limited to the <template> element, but can instead be used on any element or component in the slot.

Reply:
Yes you can add to either a <template> tag or any other tag

Reply:
I see.
I think `vueVersion < 2.5` is not necessary.